pykusto/kql_converters.py

```diff
@@ -28,7 +29,15 @@ def timedelta_to_kql(td: timedelta) -> KQL:
 
 @kql_converter(Mapping, List, Tuple)
 def dynamic_to_kql(d: Union[Mapping, List, Tuple]) -> KQL:
-    query = list(json.dumps(d))
+    try:
+        query = list(json.dumps(d))
+    except TypeError:
```

Comment:
I don't like this. I suggest the following explicit syntax:
.extend(d=f.pack(Name=col.name, Roled=f.pack_array(col.role1, col.role2, col.role3)))

Reply:
After discussion with @amos-rimon - we decided this is acceptable.